com.visualstudio.code-oss.yml

```diff
@@ -0,0 +1,124 @@
+id: com.visualstudio.code-oss
```

Comment:
There was a suggestion to use ID `io.github.microsoft.VSCode`. Maybe it would be better?